README.rst

```diff
@@ -2813,27 +2813,29 @@ Allows to specify a ``list`` of tuples for adding commands as described in
 OPENWISP_CONTROLLER_ORGANIZATION_ENABLED_COMMANDS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+--------------+------------------------------------------------------------+
-| **type**:    | ``dict``                                                   |
-+--------------+------------------------------------------------------------+
-| **default**: | .. code-block:: python                                     |
-|              |                                                            |
-|              |    {                                                       |
-|              |         '__all__': ('custom', 'reboot', 'change_password') |
-|              |    }                                                       |
-|              |                                                            |
-+--------------+------------------------------------------------------------+
++--------------+------------------------------------------------+
+| **type**:    | ``dict``                                       |
++--------------+------------------------------------------------+
+| **default**: | .. code-block:: python                         |
+|              |                                                |
+|              |    {                                           |
+|              |         # By default all commands are allowed  |
+|              |         '__all__': '*',                        |
+|              |    }                                           |
+|              |                                                |
++--------------+------------------------------------------------+
 
 Allows specifying a tuple of commands enabled for an organization. The value of
 ``__all__`` key is used if enabled commands are not specified for an organization.
 Configure the setting as follows:
```

Comment:
This setting controls the command types that are enabled on the system
By default, all command types are enabled to all the organizations,
but it's possible to disable a specific command for a specific organization
as shown in the following example: